pom.xml

```diff
@@ -39,6 +39,63 @@
         <module>gui</module>
     </modules>
 
+    <profiles>
+        <profile>
+            <id>wildfly800</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules>
+                <module>testsuite</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>wildfly810</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules>
+                <module>testsuite</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>wildfly821</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules>
+                <module>testsuite</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>wildfly902</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules>
+                <module>testsuite</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>wildfly1000</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules>
+                <module>testsuite</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>wildfly1010</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules>
+                <module>testsuite</module>
+            </modules>
+        </profile>
+    </profiles>
+
```

Comment:
Sorry, what's the point of having multiple profiles, all enabled by default and all doing the same thing (enabling testsuite module) ?